Plugin/resources/de/mobilej/plugin/adc/locales.txt

```diff
@@ -1,5 +1,7 @@
 de_DE [German (Germany)]
 en_US [English (United States)]
+en_XA [English (Pseudo Localisation ltr)]
+ar_XB [English (Pseudo Localisation rtl]
```

Comment:
@luserdroid  please add a closing ")" before the "]"

Reply:
Done, thanks for the review.